src/components/Dashboard/TodoList.vue

```diff
@@ -198,7 +198,7 @@
 
                 <!-- draft continuation in -->
                 <div
-                  v-else-if="isStatusDraft(item) && EnumUtilities.isTypeContinuationIn(item)"
+                  v-else-if="isStatusDraft(item) && isTypeContinuationIn(item)"
```

Comment:
Both the mixin and EnumUtilities methods do the same thing. I changed this here to be consistent with code on lines 177, 185 and 193, above.

Eventually, later, we should completely replace the mixin with the utilities class.